Legacy/bonej/src/main/java/org/bonej/plugins/ParticleCounter.java

```diff
@@ -570,7 +573,11 @@ private static long countDuplicates(final int[] counter,
 		minimiseLutArray(lut);
 		// all values should be seen only once,
 		// count how many >1s there are.
-		return Arrays.stream(counter).filter(i -> i > 1).count();
+		int count = 0;
+		for (int i = 1; i < l; i++) {
+			if (counter[i] > 1) count++;
+		}
+		return count;
```

Comment:
I fail to see how the for loop does anything different from the stream variant? Maybe my ignorance.

Reply:
Not sure as I'm still learning about streams. So, maybe my ignorance.

My main concern about it is that the for loop has i, l and 1, which I think might have been mixed up in the stream implementation. For me the for loop is much easier to read and understand.

Reply:
I think there may have been a couple of things going on in the commit that broke it, and haven't much interest in pursuing it beyond putting it back how it was before when it was working.

Reply:
OK, fair enough

Reply:
Hey guys, may I asked that if I remove and re-install the BoneJ now, would my CT images work? Well I achieve these CT images from multi-phase flow in the glass beads, and I'm not sure about what's going on with my data. I have many stacks which work well with BoneJ, while only in this team of experiment with this stack and its follow-up experimental images failed....

Reply:
So exciting about new releases for short tests. Thanks so much for your help!

Reply:
@alessandrofelder You can do Arrays.stream(arr).skip(1)... to start from 1. But anyhow this was carelessness on my part.